### ISSUE TITLE
Allow to send a directCommand to the Ocean Node, for testing purposes

### ISSUE THREAD

**Issue by paulo-ocean** [oceanprotocol/ocean-cli#75]
We still don't have any support for sending a `directCommand` to the node. This could be very useful for testing many features.
The idea is to have a new command on the CLI (the directCommand), where we just specify the input from a JSON file

**Comment by jamiehewitt15** [comment]
> This could be very useful for testing many features.

I'm curious about what features would use this? For testing the direct command, we can just use postman...

**Comment by paulo-ocean** [comment]
> > This could be very useful for testing many features.
> 
> I'm curious about what features would use this? For testing the direct command, we can just use postman...

Not sure what you mean by "features".. Its for testing purposes
I know we can use postman, for the direct commands, but we have to recreate/ copy paste the json all the time we want to test a specific command... If we had some command JSON samples on the cli (like we have for the DDOs on the metadata folder) we could simply run a specific command... and it would just pick the corresponding  JSON payload.
but If nobody else sees value on this, we can close it.. no worries

**Comment by jamiehewitt15** [comment]
> Not sure what you mean by "features".. It's for testing purposes

Fair enough, I miss read your comment, I thought you mean it would be used by features.

> but If nobody else sees value on this, we can close it.. no worries

I'm fine with testing via postman but maybe others have a different opinion. We can discuss at the sprint planning.

**Comment by giurgiur99** [comment]
From my side, testing with postman should be enough. What do you think @alexcos20?
